app/App.hs

```diff
@@ -6,7 +6,9 @@ import GlobalOptions
 import Juvix.Compiler.Concrete.Translation.FromParsed.Analysis.PathResolver
 import Juvix.Compiler.Pipeline
 import Juvix.Data.Error qualified as Error
-import Juvix.Prelude.Pretty hiding (Doc)
+import Juvix.Prelude.Pretty hiding
+  ( Doc,
+  )
```

Comment:
revert this change. less diff as much as we can.

Reply:
I am pretty sure this is ormolu, but let me double check

Reply:
Yes, this is the result of the `make ormolu` command for me

Reply:
Which version of ormolu are you running? The CI runs "ormolu 0.5.2.0 HEAD 810eb7839cbf4e358821447e8f4603955999e62d using ghc-lib-parser 9.4.2.20220822"

Reply:
I'm using `0.5.2.0` too:
<img width="286" alt="Screenshot 2023-04-27 at 12 44 19" src="https://user-images.githubusercontent.com/8126674/234852331-68423a45-2d94-45f8-aea2-dea3096c2359.png">